### ISSUE TITLE
no_interpolation_in_single_quotes false positive on double quoted string inside single quoted string

### ISSUE THREAD

**Issue by mnquintana** [clutchski/coffeelint#460]
Here's a test case:

```coffee
baz = 'foo = "a #{bar}"'
```

In this case `bar` is not being interpolated, but is rather just a literal string. `no_interpolation_in_single_quotes` still throws an error here though.

**Comment by lydell** [comment]
That’s the point of the rule. The idea is to prevent errors: `'#{a}'` is most likely a mistake where you probably meant `"#{a}"`. You have to explicitly show that you don’t mean interpolation: `'#\{a}'`.
